specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-05-01/integrationRuntime.json

```diff
@@ -1166,6 +1166,58 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/outboundNetworkDependenciesEndpoints": {
+      "get": {
+        "tags": [
+          "integrationRuntimes"
+        ],
+        "operationId": "IntegrationRuntimes_OutboundNetworkDependenciesEndpoints",
```

Comment:
>IntegrationRuntimes_OutboundNetworkDependenciesEndpoints [](http://example.com/codeflow?start=24&length=56)

IntegrationRuntimes_GetOutboundNetworkDependenciesEndpoints

Reply:
The operation id will be used in SDK generation. The string after "_" will become SDK method name. It should start with a Verb